src/schemas/hydroshare/uischema.json

```diff
@@ -16,579 +16,8 @@
             "multi": true,
             "trim": true
           }
-        },
-        {
-          "type": "Control",
-          "scope": "#/properties/subjects"
```

Comment:
Tagging @Maurier for awareness...
For **edit mode**, @MSDrao is removing all except 2 of the fields from the UI schema.
But for **view mode**, @Maurier is going to need all of the fields.
I guess we will just use 2 different ui schemas? On last week's call I thought I remembered Mauriel saying that we would only use a single UI schema for both edit and view...